### PR TITLE
refactor(interactive): Initiate Compiler Service with Yaml Configuration

### PR DESCRIPTION
**Pull request by shirly121**
## What do these changes do?
as titled.

## Related issue number

Fixes

### CODE REVIEW

File: interactive_engine/compiler/src/test/resources/config/gs_interactive.yaml [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+  workspace: .
+  subdirs:
+    data: target/test-classes/config  # by default data, relative to ${workspace}
+    conf: conf # by default conf, relative to ${workspace}
+    logs: logs  # by default logs, relative to ${workspace}
+log_level: INFO # default INFO
+default_graph: modern  # configure the graph to be loaded while starting the service, if graph name not specified
+# may include other configuration items of other engines
+compute_engine:
+  type: pegasus  # [hiactor|pegasus] hiactor is for high-qps scenario, pegasus is for bi scenario
```
Comment (longbinlai) [inline interactive_engine/compiler/src/test/resources/config/gs_interactive.yaml:13]: type is `hiactor`?

File: interactive_engine/compiler/conf/ir.compiler.properties [hunk 2]
```diff
@@ -16,10 +16,16 @@ pegasus.hosts: localhost:1234
 # graph.schema
 graph.schema: ../executor/ir/core/resource/modern_schema.json
 graph.store: exp
-graph.planner: {"isOn":true,"opt":"RBO","rules":["FilterMatchRule"]}
 
-# set stored procedures uri
-# graph.stored.procedures.uri: <your stored procedures path in uri format>
+graph.planner.is.on: true
```
[the remark this answers is not in the document]
Comment (shirly121) [reply interactive_engine/compiler/conf/ir.compiler.properties:20]: 这里是properties入口的配置项，是不是可以稍微详细一些？`gs_interactive.yaml`因为本身就是hierarchical结构，可以直接用`planner`作为key

File: interactive_engine/compiler/src/test/resources/config/gs_interactive.yaml (hunk 1 above)
Comment (shirly121) [reply interactive_engine/compiler/src/test/resources/config/gs_interactive.yaml]: fix in https://github.com/alibaba/GraphScope/pull/3141/commits/8d266975627bc0cda07c364bb0b0644358fc700d

**Review by longbinlai — APPROVED** [review]

**Review by zhanglei1949 — APPROVED** [review]
LGTM

**Pull request merged**